src/main/kotlin/no/nav/syfo/application/database/Database.kt

```diff
@@ -28,7 +28,7 @@ class Database(
             maximumPoolSize = config.poolSize
             minimumIdle = 1
             isAutoCommit = false
-            transactionIsolation = "TRANSACTION_REPEATABLE_READ"
+            transactionIsolation = "TRANSACTION_READ_COMMITTED"
```

Comment:
Blir det riktig da? Kan ikke en verdi bli overskrevet basert på gammel informasjon (hvis man leser en verdi, gjør en utregning basert på det, og så oppdaterer?).

Reply:
Ja, det finnes antakelig hjørnetilfeller i teorien, men jeg kan ikke se for meg noe i isdialogmelding som vil kunne bli feil på grunn av dette. Får vurdere for hvert enkelt app hvis vi tar dette videre. Men i første omgang kan vi se om samtidighetskonfliktene i isdialogmelding forsvinner.